src/main/java/uk/gov/hmcts/reform/civil/service/docmosis/sdo/SdoGeneratorService.java

```diff
@@ -333,7 +345,12 @@ private SdoDocumentFormFast getTemplateDataFast(CaseData caseData, String judgeN
                     SdoHelper.hasFastTrackVariable(caseData, "sdoR2FastTrackUseOfWelshToggle"))
                 .welshLanguageDescription(caseData.getSdoR2FastTrackUseOfWelshLanguage() != null
                                               ? caseData.getSdoR2FastTrackUseOfWelshLanguage().getDescription() : null);
+            sdoDocumentFormBuilder.sdoR2WitnessesOfFact(caseData.getSdoR2FastTrackWitnessOfFact());
+            sdoDocumentFormBuilder.sdoR2FastTrackCreditHire(caseData.getSdoR2FastTrackCreditHire());
+        } else {
+            sdoDocumentFormBuilder.fastTrackWitnessOfFact(caseData.getFastTrackWitnessOfFact());
```

Comment:
I would add .fastTrackCreditHire(caseData.getFastTrackCreditHire()) here. What do you think?